### PR TITLE
Markdown changes

### PR DESCRIPTION
**Pull request by nsehwan**
*Issue*: Latex Equations in this tutorial don't render properly in Github flavored markdown and appear as skeletal equations on the Github page, example `$$\left|0,0, ...\right> \rightarrow \left|\mathrm{GHZ}\right> = \frac{1}{\sqrt{2}}\left(\left|0,0, ...\right> + \left|1,1, ...\right>\right).$$` [here](https://github.com/aws/amazon-braket-examples/blob/main/examples/getting_started/1_Running_quantum_circuits_on_simulators.ipynb).

This is because of the incorrect usage of `\left` and `\right` markers when the delimiters following them are different (read more in the docs [here](https://latexref.xyz/_005cleft-_0026-_005cright.html)). Also Latex needs the number of `\left` and `\right` markers in any single line to be equal, otherwise it creates inconsistencies, to avoid which we need to use `\bigl` and `\bigr`.

*Description of changes:* 
Have inserted a period `.` for `\right` markets wherever we use the ket notation. Have also replaced `\left` and `\right`  with `\bigl` and `\bigr` where the latex equation in the markdown is spread over multiple   @#lines.

Using the above changes, the updated `.ipynb` file on Github represents the actual equations. Also the Jupyter notebook continues to represent the actual equations as before.

### CODE REVIEW

File: examples/getting_started/1_Running_quantum_circuits_on_simulators.ipynb [hunk 1]
```diff
@@ -67,7 +67,7 @@
    "metadata": {},
    "source": [
     "Goal: Prepare an $N$-qubit GHZ state: \n",
-    "$$\\left|0,0, ...\\right> \\rightarrow \\left|\\mathrm{GHZ}\\right> = \\frac{1}{\\sqrt{2}}\\left(\\left|0,0, ...\\right> + \\left|1,1, ...\\right>\\right).$$"
+    "$$\\left|0,0, ...\\right.> \\rightarrow \\left|\\mathrm{GHZ}\\right.> = \\frac{1}{\\sqrt{2}}\\left(\\left|0,0, ...\\right.> + \\left|1,1, ...\\right.>\\right).$$"
```
Comment (speller26) [inline examples/getting_started/1_Running_quantum_circuits_on_simulators.ipynb:70]: The correct angle bracket is `\rangle`

```suggestion
    "$$\\left|0,0, ...\\rangle \\rightarrow \\left|\\mathrm{GHZ}\\rangle = \\frac{1}{\\sqrt{2}}\\left(\\left|0,0, ...\\rangle + \\left|1,1, ...\\right.>\\right).$$"
```

File: examples/getting_started/1_Running_quantum_circuits_on_simulators.ipynb [hunk 2]
```diff
@@ -257,9 +257,9 @@
     "The example code below outputs the state vector, the expectation value of $Z\\otimes Z\\otimes Z$, and the amplitude of the $|111\\rangle$ state of a three-qubit GHZ state. \n",
     "\n",
     "To reiterate, the following output is expected:\n",
-    "$$\\left|\\mathrm{GHZ}\\right> = \\frac{1}{\\sqrt{2}}\\left(\\left|0,0,0\\right> + \\left|1,1,1\\right>\\right) = \\left[\\frac{1}{\\sqrt{2}},0,0,0,0,0,0,\\frac{1}{\\sqrt{2}}\\right],$$\n",
+    "$$\\left|\\mathrm{GHZ}\\right.> = \\frac{1}{\\sqrt{2}}\\left(\\left|0,0,0\\right.> + \\left|1,1,1\\right.>\\right) = \\bigl[\\frac{1}{\\sqrt{2}},0,0,0,0,0,0,\\frac{1}{\\sqrt{2}}\\bigr],$$\n",
```
Comment (speller26) [inline examples/getting_started/1_Running_quantum_circuits_on_simulators.ipynb:260]: ditto

File: examples/getting_started/1_Running_quantum_circuits_on_simulators.ipynb [hunk 3]
```diff
@@ -257,9 +257,9 @@
...
     "\n",
-    "for which $\\left<ZZZ\\right>=0$ and $\\left<111|\\mathrm{GHZ}\\right>=\\frac{1}{\\sqrt{2}}$."
+    "for which $ <ZZZ>=0$ and $<111|\\mathrm{GHZ}>=\\frac{1}{\\sqrt{2}} $."
```
Comment (speller26) [inline examples/getting_started/1_Running_quantum_circuits_on_simulators.ipynb:262]:
```suggestion
    "for which $ \\langle ZZZ \\rangle = 0$ and $\\langle 111 | \\mathrm{GHZ} \\rangle = \\frac{1}{\\sqrt{2}} $."
```

File: examples/getting_started/1_Running_quantum_circuits_on_simulators.ipynb (hunk 2 above)
Comment (speller26) [reply examples/getting_started/1_Running_quantum_circuits_on_simulators.ipynb:260]:
```suggestion
    "$$\\left|\\mathrm{GHZ}\\rangle = \\frac{1}{\\sqrt{2}}\\left(\\left|0,0,0\\rangle + \\left|1,1,1\\rangle\\right) = \\bigl[\\frac{1}{\\sqrt{2}},0,0,0,0,0,0,\\frac{1}{\\sqrt{2}}\\bigr],$$\n",
```

**Comment by ajberdy** [comment]
Thanks for the contribution! This change has gone stale with our current notebooks so we are closing this PR

**Pull request closed without merging**